lib/msf/core/exploit/sqli/sqlitei/common.rb

```diff
@@ -217,7 +217,7 @@ def blind_detect_length(query, timebased)
     #  @param query [String] the SQL query to execute
     #  @param length [Integer] the expected length of the result
     #  @param known_bits [Integer] (returned by get_bitmask) bits that are common to all the output characters
-    #  @param bits_to_guess [Integer] (returned by get_bitmask) The number of bits to guess on each character of the output
+    #  @param _bits_to_guess [Integer] (returned by get_bitmask) The number of bits to guess on each character of the output
```

Comment:
```suggestion
    #  @param bits_to_guess [Integer] (returned by get_bitmask) The number of bits to guess on each character of the output
```

`bits_to_guess` is this method's public API - we just prefix the argument name with `_` to indicate to the developer that the implementation doesn't use the argument, as required by rubocop - https://docs.rubocop.org/rubocop/cops_lint.html#lintunusedmethodargument.

I don't know if YARD has first class support for this scenario or not - but I'm guessing not as they seem to opt out of this linting rule on a per-method basis https://github.com/lsegal/yard/blob/359006641260eef1fe6d28f5c43c7c98d40f257d/lib/yard/handlers/base.rb#L205

Reply:
If we don't update this, then YARD will not be able to find the parameter name and will not be able to generate the documentation appropriately. Instead it will state that it can't find a parameter with that name in the function definition to document.

So yes whilst I would like it to recognize this, at the moment this is not the case and it is preventing us from generating proper documentation in these scenarios.

Seeing as its just a small addition to what is already there I don't see the harm in adding the `_` to the YARD definitions so that it can find the appropriate parameter and document it accordingly.